change wording in tutorial (#8303)
withastro/docs: src/content/docs/en/tutorial/4-layouts/1.mdx

- Let the layout be responsible for the HTML rendering of any common elements.

> - Delete anything from each page that is no longer responsible for rendering, because it is being handled by the layout, including: 

Reviewer: ```suggestion
- Delete anything from each individual page that is now being handled by the layout, including: 
```
In fact, the original is grammatically correct, but, I agree that it's an awkward sentence to read (feels more natural when spoken) so let's change it!

I think maybe we can make this even better. What do you think about something like this?? Does this feel *too* short, or does this still get the idea across?
